aderyn_driver/src/process_auto.rs

```diff
@@ -48,20 +60,34 @@ pub fn with_project_root_at(
 
     // dbg!(&compilation_groups);
 
+    let mut remappings = vec![];
+    if let Some(custom_remappings) = read_remappings(root_path) {
+        remappings.extend(custom_remappings);
+        remappings.dedup();
+    }
+
     let mut workspace_contexts: Vec<WorkspaceContext> = vec![];
 
     for (version, file_paths) in &compilation_groups {
         let solc = Solc::find_or_install_svm_version(version).unwrap();
         let solc_bin = solc.solc.to_str().unwrap();
 
         let command = Command::new(solc_bin)
+            .args(remappings.clone())
             .arg("--ast-compact-json")
             .args(file_paths)
-            .current_dir("/")
+            .current_dir(root_path)
             .stdout(Stdio::piped())
```

Comment:
We migrated away from `/`! This allowed for remappings to be slightly easy at the cost of not allowing the user to open vscode anywhere.  Which we will tackle later as discussed  : )